nats.go

```diff
@@ -1166,6 +1173,13 @@ func UserInfo(user, password string) Option {
 	}
 }
 
+func UserInfoHandler(cb UserInfoCB) Option {
```

Comment:
the name `UserInfo` is a bit ambigous. 
Maybe something hinting it's about Username and password being set?

Reply:
Ok, understood that its done for consistency.